public/src/components/Main.tsx

```diff
@@ -106,6 +109,7 @@ export class Main extends React.Component<MainProps, MainState> {
             stockDetailsMap: this.props.stockDetailsMap,
             stocksReservedMap: this.props.stocksReservedMap,
             stockBriefInfoMap: stockBriefInfoMap,
+            isPhoneVerified: this.props.user.getIsPhoneVerified(),
```

Comment:
Pass `isPhoneVerified` as a prop from App.tsx instead of declaring it as a state variable again.